<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->












<h2>Gene Expression Omnibus (GEO)</h2>

<p>The Gene Expression Omnibus (GEO) is a publicly-funded repository of gene-expression studies maintained by the NCBI. Access the repository <a href="http://www.ncbi.nlm.nih.gov/geo/">here</a>.</p>

<h3>Using the GEO web interface to identify differentially expressed genes in gene-expression datasets</h3>

<p>See the video tutorial <a href="https://www.youtube.com/watch?v=EUPmGWS8ik0" target="_blank">here</a></p>
<ul>
<li>Select a data set
<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44097" target="_blank">e.g. GSE44097</a>
<li>Click 'Analyze with GEO2R</a>'
<li>Define groups of samples (e.g. treated and control)</li>
<li>Assign samples to groups</li>
<li>Click 'Top 250' to give you list of differentially expressed genes</li>
</ul>














































