<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<html><head>
<meta http-equiv="content-type" content="text/html; charset=windows-1252">
<link rel="stylesheet" type="text/css" href="geo-case-studies2_files/main.css">
</head>

 <body><nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/using-search-engines.html">Using MS search engines</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/biological-interpretation.html">Network and functional analyses</a>
    </li><li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/repositories-and-tools.html">Links to online resources and tools</a>
    </li></ul>



    </li><li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/label-free-studies.html">Label-free quantification</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </li></ul>
    



    

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->


<h2>Identification and analysis of  differentially expressed genes in gene-expression datasets</h3>
<p>The Gene Expression Omnibus (GEO) is a publicly-funded repository of gene-expression studies maintained by the NCBI. Access the repository <a href="http://www.ncbi.nlm.nih.gov/geo/">here</a>. Example GEO datasets are in the table below</p>

<p>
Perform these tasks to familiarize yourself with respect to basis analyses 
of this type of data. (Note that some of the tasks can be performed using the GEO web interface - see below. However, for the purposes of the workshop perform these analyses yourself in R)</p>
<ul>
  <li>Select a GEO Dataset, and follow the links to read about the study</li>
  <ul>
    <li>Make sure to understand the study and the experimental design</li>
    <ul>
    <li>What is the objective of the study?</li>
    <li>What platform has been used and what has been measured?</li>
    <li>What are the sample groups?</li>
    <li>How many replicates have been performed, and what type of replicates are they?</li>
    </ul>
  </ul>
  <li>Data analyses and visualizations</li>
  <ul>
    <li>(use R code for the analysis)
    <li>Visualize the distributions of the data (for example using histograms or boxplots)</li>
    <li>Does the data need to be normalized or scaled?</li>
    <li>Visualize the correlation between samples (for example using pairwise scatterplots and computing correlations)</li>
    <li>Use PCA to visualize the structure in the data. Do the sample groups cluster?</li>
    <li>Compute the ratio (use the log2 ratio) for pairwise comparison(s) within the dataset</li>
    <li>Identify DEGs within the dataset (use T-test)</li>
    <ul>
      <li>Note that for the purposes of this workshop we do not do any multiple testing correction</li> 
    </ul>
    <li>Plot Volcano plot(s) using the ratio and T-test p-values</li>
  </ul>
  <li>Biological interpretation</li>
  <ul>
    <li>How many DEGs are there with e.g. p-value < 0.05?</li>
    <li>Identify functional gene networks using STRING</li>						    
    <li>Identify enriched pathways and processes using e.g. GOrilla or Enrichr</li>
    <li>How does your biological interpretation relate to the original study? What could you add to the analysis?</li>
  </ul>

</p>





<h2>Case Studies (GEO Datasets) from GEO</h2>


<table border="1" cellspacing="1" cellpadding="0">
<tbody>
<tr>
<td><b>Study Title</b></td>
<td><b>GEO Link</b></td>
<td><b>Data</b></td>
</tr>

<tr>
<td>PTEN deletion mutation effect on colon cancer cells</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2446" target="_blank">GDS2446</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS2446.csv">mat.GDS2446.csv</a></td>
</tr>

<tr>
<td>Colorectal adenoma formation</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2947" target="_blank">GDS2947</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS2947.csv">mat.GDS2947.csv</a></td>
</tr>

<tr>
<td>Beta-catenin inactivation effect on intestinal crypts</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2984" target="_blank">GDS2984</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS2984.csv">mat.GDS2984.csv</a></td>
</tr>

<tr>
<td>Beta-catenin deficiency model: kidney</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS3322" target="_blank">GDS3322</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS3322.csv">mat.GDS3322.csv</a></td>
</tr>

<tr>
<td>Colon carcinoma response to butyrate and aspirin</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS332" target="_blank">GDS332</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS332.csv">mat.GDS332.csv</a></td>
</tr>

<tr>
<td>Beta-catenin depletion effect on pancreatic cancer cell line</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS3578" target="_blank">GDS3578</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS3578.csv">mat.GDS3578.csv</a></td>
</tr>

<tr>
<td>Colorectal cancer tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4382" target="_blank">GDS4382</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4382.csv">mat.GDS4382.csv</a></td>
</tr>

<tr>
<td>Colorectal cancer tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4379" target="_blank">GDS4379</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4379.csv">mat.GDS4379.csv</a></td>
</tr>

<tr>
<td>Ls174T colon cancer cell line response to Wnt signaling inhibition</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4386" target="_blank">GDS4386</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4386.csv">mat.GDS4386.csv</a></td>
</tr>

<tr>
<td>5-aza-2<u+2019>-deoxycytidine effect on colorectal cancer cell lines</u+2019></td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4397" target="_blank">GDS4397</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4397.csv">mat.GDS4397.csv</a></td>
</tr>

<tr>
<td>Model of <u+03b2>-catenin overexpression in embryonic kidney</u+03b2></td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4449" target="_blank">GDS4449</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4449.csv">mat.GDS4449.csv</a></td>
</tr>

<tr>
<td>Sporadic medulloblastomas</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4469" target="_blank">GDS4469</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4469.csv">mat.GDS4469.csv</a></td>
</tr>

<tr>
<td>Medulloblastomas in children</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4471" target="_blank">GDS4471</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4471.csv">mat.GDS4471.csv</a></td>
</tr>

<tr>
<td>MYC-driven medulloblastoma model</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4478" target="_blank">GDS4478</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4478.csv">mat.GDS4478.csv</a></td>
</tr>

<tr>
<td>Colorectal cancer cell line SW480 response to Snail overexpression</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4596" target="_blank">GDS4596</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4596.csv">mat.GDS4596.csv</a></td>
</tr>

<tr>
<td>Central nervous system primitive neuroectodermal tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4838" target="_blank">GDS4838</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS4838.csv">mat.GDS4838.csv</a></td>
</tr>


<tr>
<td>Beta-catenin depletion effect on pancreatic cancer cell line</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS5324" target="_blank">GDS5324</a></td>
<td><a href="https://alt-5f16d636b7ae3.blackboard.com/bbcswebdav/pid-7398469-dt-content-rid-34059639_1/courses/BIOL3063-34495-25-26/BIOL3063-34495-25-26_ImportedContent_20250715014911/omicsanalysistutorial/data/mat.GDS5324.csv">mat.GDS5324.csv</a></td>
</tr>

</tbody>
</table>


<p><p>

<h2>Using the GEO web interface to identify differentially expressed genes in gene-expression datasets</h3>

<p>See the video tutorial <a href="https://www.youtube.com/watch?v=EUPmGWS8ik0" target="_blank">here</a></p>
<ul>
<li>Select a data set
<ul>
<li><span class="WYSIWYG_LINK">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44097</span></li>
</ul>
</li>
<li>Click 'Analyze with GEO2R</a>'
<ul>
<li><span class="WYSIWYG_LINK">http://www.ncbi.nlm.nih.gov/geo/geo2r/?acc=GSE44097</span></li>
</ul>
</li>
<li>Define groups of samples (e.g. treated and control)</li>
<li>Assign samples to groups</li>
<li>Click 'Top 250' to give you list of differentially expressed genes</li>
</ul>



















































</p></body></html>
