<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->


























<h2>Case Studies (GEO Datasets) from GEO</h2>

<p>
  
<p>Here we make use of the Gene Expression Omnibus (GEO) to provide example transcriptomics datasets for analysis. Each of the entries in the table below is a GEO Dataset - a curated Gene Expression study. Here are some tasks to familiarize yourself with respect to basis analyses of this type of data:

  <ul>
  <li>Select a GEO Dataset</li>
  <li>For each GEO Dataset, you can work with the online tools provided by GEO for analysis, or your can work with the raw data provided in the links below</li>
  <li>If using the online tools, you will see that you can explore the clustered gene-expression profiles associated with each GEO Dataset. By clicking on the GEO Series link you can also perform differential gene expression analysis using the 'Analyze with GEO2R' button</li>
  <li>To work with the raw data, each of the CSV files listed in the table is a matrix (gene x sample) of gene-expression data.</li>
  <ul>
    <li>From the sample descriptions, and the experimental design described on the GEO Dataset web-page, select a group of samples to perform differential gene-expression analysis (for example this could be comparing control vs treated samples). Note that this may be a subset (or all of the samples in the file)</li> 
    <li>For each gene, calculate a fold-change (ratio) ratio between your sample groups (e.g. treated/control) and express this as logarithm base 2</li>
    <li>For each gene, you could also calculate a p-value (e.g. T-test) to test whether your sample groups are statistically significantly different</li>
    <li>Sort the data according to the calculated ratio (and/or p-value), and identify those genes where log2 ratio &gt; 1 or log 2 ratio &lt; -1 or p &lt; 0.05</li>
    <li>For the differential genes, use Gene Set Enrichment and/or functonal network analysis to uncover potential pathways or processes that are represented.</li>
  </ul>
  </ul>
<p>
<p>

<table border="1" cellspacing="1" cellpadding="0">
<tbody>
<tr>
<td><b>Study Title</b></td>
<td><b>GEO Link</b></td>
<td><b>Data</b></td>
</tr>

<tr>
<td>PTEN deletion mutation effect on colon cancer cells</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2446" target="_blank">GDS2446</a></td>
<td><a href="data/mat.GDS2446.csv">mat.GDS2446.csv</a></td>
</tr>

<tr>
<td>Colorectal adenoma formation</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2947" target="_blank">GDS2947</a></td>
<td><a href="data/mat.GDS2947.csv">mat.GDS2947.csv</a></td>
</tr>

<tr>
<td>Beta-catenin inactivation effect on intestinal crypts</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2984" target="_blank">GDS2984</a></td>
<td><a href="data/mat.GDS2984.csv">mat.GDS2984.csv</a></td>
</tr>

<tr>
<td>Beta-catenin deficiency model: kidney</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS3322" target="_blank">GDS3322</a></td>
<td><a href="data/mat.GDS3322.csv">mat.GDS3322.csv</a></td>
</tr>

<tr>
<td>Colon carcinoma response to butyrate and aspirin</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS332" target="_blank">GDS332</a></td>
<td><a href="data/mat.GDS332.csv">mat.GDS332.csv</a></td>
</tr>

<tr>
<td>Beta-catenin depletion effect on pancreatic cancer cell line</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS3578" target="_blank">GDS3578</a></td>
<td><a href="data/mat.GDS3578.csv">mat.GDS3578.csv</a></td>
</tr>

<tr>
<td>Colorectal cancer tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4382" target="_blank">GDS4382</a></td>
<td><a href="data/mat.GDS4382.csv">mat.GDS4382.csv</a></td>
</tr>

<tr>
<td>Colorectal cancer tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4379" target="_blank">GDS4379</a></td>
<td><a href="data/mat.GDS4379.csv">mat.GDS4379.csv</a></td>
</tr>

<tr>
<td>Ls174T colon cancer cell line response to Wnt signaling inhibition</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4386" target="_blank">GDS4386</a></td>
<td><a href="data/mat.GDS4386.csv">mat.GDS4386.csv</a></td>
</tr>

<tr>
<td>5-aza-2<U+2019>-deoxycytidine effect on colorectal cancer cell lines</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4397" target="_blank">GDS4397</a></td>
<td><a href="data/mat.GDS4397.csv">mat.GDS4397.csv</a></td>
</tr>

<tr>
<td>Model of <U+03B2>-catenin overexpression in embryonic kidney</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4449" target="_blank">GDS4449</a></td>
<td><a href="data/mat.GDS4449.csv">mat.GDS4449.csv</a></td>
</tr>

<tr>
<td>Sporadic medulloblastomas</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4469" target="_blank">GDS4469</a></td>
<td><a href="data/mat.GDS4469.csv">mat.GDS4469.csv</a></td>
</tr>

<tr>
<td>Medulloblastomas in children</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4471" target="_blank">GDS4471</a></td>
<td><a href="data/mat.GDS4471.csv">mat.GDS4471.csv</a></td>
</tr>

<tr>
<td>MYC-driven medulloblastoma model</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4478" target="_blank">GDS4478</a></td>
<td><a href="data/mat.GDS4478.csv">mat.GDS4478.csv</a></td>
</tr>

<tr>
<td>Colorectal cancer cell line SW480 response to Snail overexpression</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4596" target="_blank">GDS4596</a></td>
<td><a href="data/mat.GDS4596.csv">mat.GDS4596.csv</a></td>
</tr>

<tr>
<td>Central nervous system primitive neuroectodermal tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4838" target="_blank">GDS4838</a></td>
<td><a href="data/mat.GDS4838.csv">mat.GDS4838.csv</a></td>
</tr>


<tr>
<td>Beta-catenin depletion effect on pancreatic cancer cell line</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS5324" target="_blank">GDS5324</a></td>
<td><a href="data/mat.GDS5324.csv">mat.GDS5324.csv</a></td>
</tr>

</tbody>
</table>





















































