<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="overview.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="ms-study-6.html">Embryonic stem cell proteomes</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->










































<h2>Case Studies (GEO Datasets) from GEO</h2>

<table border="1" cellspacing="1" cellpadding="0">
<tbody>
<tr>
<td><b>Study Title</b></td>
<td><b>GEO Link</b></td>
<td><b>Data</b></td>
<td><b>Notes</b></td>
<td><b>Tasks</b></td>
</tr>

<tr>
<td>PTEN deletion mutation effect on colon cancer cells</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2446" target="_blank">GDS2446</a></td>
<td><a href="data/mat.GDS2446.csv">mat.GDS2446.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Colorectal adenoma formation</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2947" target="_blank">GDS2947</a></td>
<td><a href="data/mat.GDS2947.csv">mat.GDS2947.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Beta-catenin inactivation effect on intestinal crypts</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS2984" target="_blank">GDS2984</a></td>
<td><a href="data/mat.GDS2984.csv">mat.GDS2984.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Beta-catenin deficiency model: kidney</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS3322" target="_blank">GDS3322</a></td>
<td><a href="data/mat.GDS3322.csv">mat.GDS3322.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Colon carcinoma response to butyrate and aspirin</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS332" target="_blank">GDS332</a></td>
<td><a href="data/mat.GDS332.csv">mat.GDS332.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Beta-catenin depletion effect on pancreatic cancer cell line</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS3578" target="_blank">GDS3578</a></td>
<td><a href="data/mat.GDS3578.csv">mat.GDS3578.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Colorectal cancer tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4382" target="_blank">GDS4382</a></td>
<td><a href="data/mat.GDS4382.csv">mat.GDS4382.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Colorectal cancer tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4379" target="_blank">GDS4379</a></td>
<td><a href="data/mat.GDS4379.csv">mat.GDS4379.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Ls174T colon cancer cell line response to Wnt signaling inhibition</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4386" target="_blank">GDS4386</a></td>
<td><a href="data/mat.GDS4386.csv">mat.GDS4386.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>5-aza-2<U+2019>-deoxycytidine effect on colorectal cancer cell lines</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4397" target="_blank">GDS4397</a></td>
<td><a href="data/mat.GDS4397.csv">mat.GDS4397.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Model of <U+03B2>-catenin overexpression in embryonic kidney</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4449" target="_blank">GDS4449</a></td>
<td><a href="data/mat.GDS4449.csv">mat.GDS4449.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Sporadic medulloblastomas</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4469" target="_blank">GDS4469</a></td>
<td><a href="data/mat.GDS4469.csv">mat.GDS4469.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Medulloblastomas in children</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4471" target="_blank">GDS4471</a></td>
<td><a href="data/mat.GDS4471.csv">mat.GDS4471.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>MYC-driven medulloblastoma model</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4478" target="_blank">GDS4478</a></td>
<td><a href="data/mat.GDS4478.csv">mat.GDS4478.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Colorectal cancer cell line SW480 response to Snail overexpression</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4596" target="_blank">GDS4596</a></td>
<td><a href="data/mat.GDS4596.csv">mat.GDS4596.csv</a></td>
<td></td>
<td>
</td>
</tr>

<tr>
<td>Central nervous system primitive neuroectodermal tumors</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS4838" target="_blank">GDS4838</a></td>
<td><a href="data/mat.GDS4838.csv">mat.GDS4838.csv</a></td>
<td></td>
<td>
</td>
</tr>


<tr>
<td>Beta-catenin depletion effect on pancreatic cancer cell line</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GDS5324" target="_blank">GDS5324</a></td>
<td><a href="data/mat.GDS5324.csv">mat.GDS5324.csv</a></td>
<td></td>
<td>
</td>
</tr>

</tbody>
</table>





<h2>Additional GEO Studies</h2>


<table border="1" cellspacing="1" cellpadding="0">
<tbody>
<tr>
<td><b>Study Title</b></td>
<td><b>GEO Link</b></td>
<td><b>Data</b></td>
<td><b>Notes</b></td>
<td><b>Tasks</b></td>
</tr>


<td>Gene expression in colon cancer stem cells (CSC) cultures identified by Wnt signaling levels</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33112" target="_blank">GSE33112</a></td>
<td>(obtain via GEOQuery)</td>
<td>&#160;</td>
<td>
</td>
</tr>




<tr>
<td>Targets of Wnt-1 in C57MG</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE1899" target="_blank">GSE1899</a></td>
<td><a href="data/GSE1899.matrix.txt">GSE1899.matrix.txt</a></td>
<td>Data file columns:<br />GENE_SYMBOL Mouse gene symbol corresponding to this microarray probe<br />ID_REF Microarray probe identifier<br />GSM* signal intensity columns for each sample<br />consists intensity measureof single comparison between control and treated</td>
<td>
<ul>
<li>Using Bash or R code, compute the number of different probes and the unique number of genes represented in this data</li>
<li>Using R code, calculate a ratio of treated/control and express this as logarithm base 2</li>
<li>Sort the data according to the calculated ratio, and identify those genes which have at least one probe with log2 ratio &gt; 1 or log 2 ratio &lt; -1</li>
<li>Use the DAVID tools to analyse the set of genes with log2 ratio &gt; 1 or log2 ratio &lt; -1</li>
</ul>
</td>
</tr>
<td>Gene expression in colon cancer stem cells (CSC) cultures identified by Wnt signaling levels</td>
<td><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33112" target="_blank">GSE33112</a></td>
<td>(obtain via GEOQuery)</td>
<td>&#160;</td>
<td>
<ul>
<li>Using the GEO interface, identify genes that are differentially expressed in low Colo200 vs high Colo200</li>
<li>Download the top 250 genes, and analyse using DAVID</li>
</ul>
</td>
</tr>

</tbody>
</table>














































