<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->










<h3>Aims and Objectives</h3>

<p>This tutorial is intended to introduce basic principles and commonly used tools for the analysis of omics data (princioally proteomics and transcriptomics). For example:
<ul>
  <li>How to use a mass spectrometry "search engine" to identify proteins from mass spectra in proteomics experiments
  <li>How to identify differentially expressed genes from transcriptomic studies in the Gene Expression Omnibus
  <li>Using online tools to analyze gene or protein sets in terms of their underlying functional networks
  <li>How to do basic gene set enrichment analysis
</ul>

<p>The tutorial is centred around using proteomic and transcriptomic case studies, taken from the scientific literature. Data are provided for each case study and a synopsis or link to the associated paper.

<h3>Overview and getting started</h3>

<h4>Proteomics</h4>
<ul>
<li><b>Read through the case studies, and select one for analysis</b>
<li><b>To search the mass-spectra against a sequence database:</b>
<li>Download the peak list files provided for each case study. See <a href="ms-data-peak-list-formats.html">here</a> for guidance on peak list file formats</li>
<li>Use the Mascot or Tandem search engine to identify proteins. See <a href="using-search-engines.html">here</a> for guidance</li>
<li>Run the search and extract the proteins from the output file
</ul>

<h4>Transcriptomics</h4
<ul>
  <li><b>Read through the case studies, and select one for analysis</b>
  <li>Follow the guide on using analyzing gene-expression datasets within GEO <a href="gene-expression-omnibus.html">here</a></li>  
</ul>


<h4>Functional analysis of gene or protein sets</h4>
<ul>
<li><b>To identify enriched Gene Ontology categories,</b> analyze the genes or proteins in GOrilla (See <a href="biological-interpretation.html">here</a>)
<li><b>Perform gene network analysis</b> by analyzing the genes or proteins in STRING (See <a href="biological-interpretation.html">here</a>)
</ul>

  <p>
  <p>


<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.

<p>
<p>This resource was developed with financial support to Professor Rob Ewing from The University Center for Innovation in Teaching and Education (UCITE), Case Western Reserve University, Cleveland, Ohio



















