<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="ms-study-6.html">Embryonic stem cell proteomes</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->











































<h3><strong>Overview</strong></h3>

<p>There are several different methods for quantifying peptides and proteins in shotgun proteomics experiments. Broadly, these can be divided into those techniques which label the peptides or proteins in samples to be compared (e.g. TMT, iTRAQ, SILAC) and label-free techniques. Label-free techniques are those techniques in which each sample is run as a separate mass-spectrometry run and no labeling of peptides proteins is required. Label-free techniques include 'spectral counting' in which the quantification for each peptide is determined from the number of times that fragmentation of matching spectra is observed, and techniques which measure the total ion current associated with each peptide. The latter approach is very commonly used, and on this page some example outputs from such studies are provided.</p>

<p>Typically, there are several output files associated with a label-free study. First, quantifications of all of the peptides observed in the study (Step 1). Second, by matching these peptides to proteins, quantification of all of the identified proteins in a study (Step 2). Finally, merging of the proteins from each sample across the study, so that the protein quantifications can be compared across the study (Step 3). See below for example files from each of these steps.</p>



<h3><strong>Data</strong></h3>     

<table border="1" cellpadding="5" cellspacing="0" width="100%">
  <tbody>
    <tr>
      <td valign="top">

	<p><strong>Quantification Data Examples</strong></p>


	<ul>
	  <li><a class="urllink" href="data/Peptide-Hi3-example.csv">Quantified Peptides single sample (Step 1)</a></li>
	  <li><a class="urllink" href="data/Protein-Hi3-example.csv">Quantified Proteins single sample (Step 2)</a></li>
	  <li><a class="urllink" href="data/MergedFinalProt_GSK3-study-6-samples.csv">Quantified Proteins merged multiple samples (Step 3)</a></li>
	</ul>

	</td>
   

      </tr>
    </tbody>
</table>

<h3><strong>References</strong></h3>
<p>Relative and Absolute Quantitation in Mass Spectrometry-Based Proteomics (<a class="urllink" href="https://pubmed.ncbi.nlm.nih.gov/29894226/" rel="nofollow">PubMed</a>)</p>
<p> Quantitative mass spectrometry in proteomics: a critical review (<a class="urllink" href="https://pubmed.ncbi.nlm.nih.gov/17668192/" rel="nofollow">PubMed</a>)</p>


<p>&nbsp;</p>



<
































