<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->







































<h3>Mass-spectrometry Raw Data and File Formats</h3>
<p>There are many different file formats for raw MS data, depending upon the manufacturer of the instrument. Several open, universal data formats for MS data have been developed, such as <a href="http://www.ncbi.nlm.nih.gov/pubmed/15529173" rel="nofollow" target="_blank">mzXML</a> and the newer <a href="http://www.psidev.info/index.php?q=node/80" rel="nofollow" target="_blank">mzML</a>. Typically, mass spectrometers do not generate data in these formats directly, but raw data can be converted to open formats using vendor-specific software.</p>
<h3>About mzXML Files</h3>
<p>A mzXML file is a <a href="http://en.wikipedia.org/wiki/XML" target="_blank" title="XML">XML</a>-based (eXtensible Markup Language) file that uses Base-64 representation of the mz-intensity pairs to incorporate the large volumes of generated data into the XML format.</p>
<h4>To visualize a mzXML File</h4>
<ol>
  <li>
    <ol>
      <li>For an example mzXML file, right-click <a href="data/DP00570_F02.mzxml">here</a>, and click <strong>save link as</strong>.</li>
      <li>Optional: review the mzXML and its associated <a href="http://sashimi.sourceforge.net/schema_revision/mzXML_1.1.1/mzXML.xsd" target="_blank">schema</a></li>
      <li>Open InsilicosViewer. If you have not installed InsilicosViewer, see Software Requirements in the <a href="proteomics-tutorial.html">Getting Started </a>section.</li>
      <li>Click <strong>File Open</strong>, navigate to the saved mzXML file, and click Open.</li>
      </ol>
    </li>
</ol>
<p><img alt="" height="276" src="images/insilico.png" width="400" /></p>
<ol>The two panes in the viewer represent the chromatogram (upper) and scan (lower) profiles respectively.
  <li>Place the cursor in the chromatogram pane. The corresponding scan appears in the lower pane.</li>
  <li>Toggle between the base peak and total ion chromatogram views (Tools-&gt;drawing style)
    <ul>
      <li>What do these two views represent? For information, click <a href="http://en.wikipedia.org/wiki/Mass_chromatogram" target="_blank">here</a>.</li>
      </ul>
    </li>
  <li>Find a parent ion scan and review its associated MS/MS scans.
    <ul>
      <li>For each parent ion (MS) scan, how many MS/MS scans are there?</li>
      </ul>
    </li>
</ol>

<h2>Preparing MS data for Analysis</h2>
<p>To simplify the identification process, mzXML files are often processed to simpler ASCII representations of the data, known as peak lists. There are several different peak list file formats (explained <a href="http://www.matrixscience.com/help/data_file_help.html" target="_blank">here)</a>. We will use the Mascot Generic Format (MGF) to analyze for peptides and proteins. The corresponding MGF file for the mzXML data file visualized above is <a href="data/DP00570_F02.mgf" target="_blank">here</a>.</p>
<ul>
  <li>Review the MGF file.
    <ul>
      <li>Each section between &quot;BEGIN IONS&quot; and &quot;END IONS&quot; contains spectra (mass and intensity) for the parent ion and fragment ions.</li>
      </ul>
    </li>
</ul>


<p>In the next section, we take peak list files and search sequence databases for matching peptides.</p>
<table border="0" cellpadding="0" cellspacing="0" width="90%">
  <tbody>
    <tr>
      <td bgcolor="#e3e3e3" valign="top">
	<h3>Further Analyses</h3>
	<p>To get familiar with file formats, compare the number of MS/MS spectra in an MGF file and its corresponding mZXML file.</p>
	</td>
      </tr>
    </tbody>
</table>
<p>&nbsp;</p>



















