<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="ms-study-6.html">Embryonic stem cell proteomes</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->























































































































<h3> Further Analysis of Mass-Spectrometry data</h3>
<p>The next stage is to analyze the peptides and proteins identified by the search engines. For this tutorial, we will look in detail at the peptide summary report produced by Mascot. We will also look at how modifying the search parameters impacts the results.</p>
<h3>Exploring Mascot results</h3>
<ol>
  <li>Explore the various components of the <a href="features-of-the-mascot-results.html">peptide summary</a>.</li>
  <li>Try some of the following modifications:
    <ul>
      <li>Select fixed or variable modifications (e.g, try phospho ST or phospho Y).</li>
      <li>Select the Error tolerant checkbox.</li>
      <li>Select the decoy checkbox to search "decoy" databases.</li>
      </ul>
    </li>
  <li>Explore the impacts of these modifications on the results. Are there any hits?</li>
</ol>
<h3>Exporting protein lists from a search engine</h3>
<p>In the Mascot Peptide Summary Report:</p>
<ol>
  <li>In the <strong>Format As </strong>dropdown list, select <strong>Export Search Results</strong>.</li>
  <li>Click <strong>Format As</strong>.</li>
  <li>In the <strong>Export Format</strong> dropdown list, select <strong>CSV</strong>.</li>
  <li>Ensure that the <strong>Protein Hit Information</strong> checkbox is selected, and that all other sections are deselected.</li>
  <li>Click <strong>Export Search Results.</strong></li>
  <li>Save file to appropriate location.</li>
</ol>
<p>In the X!Tandem results page:</p>
<p>1. in the <strong>display as:</strong> drop down menu, select <strong>excel</strong></p>
<h4>Note:</h4>
<p>The results may need to be reformatted (gene list extracted) or identifiers mapped to gene symbols or entrez gene IDs for use in some of the biological analysis tools.</p>
<table border="0" cellpadding="0" cellspacing="0" width="90%">
  <tbody>
    <tr>
      <td bgcolor="#e3e3e3" valign="top">
	<h3>Further Analyses</h3>
	<p>Compare results from searching the same MS data using Mascot and X!Tandem (and using the same or similar search parameters such as the database).</p>
	<ul>
	  <li>Are the search results from Mascot and X!Tandem the same?</li>
	  <li>How many proteins are in common?</li>
	  <li>Are the proteins in common ranked in the same order?</li>
	  <li>Pick a protein in common between Mascot and GPM - compare peptide coverage of this protein.</li>
	  </ul>
	</td>
      </tr>
    </tbody>
</table>
<p>&nbsp;</p>




















































