<!-- BEGIN navigation bar and head. do not edit here, use perl script to add to all html files -->
<head>
<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
</head>

 <nav id="primary_nav_wrap">
    <ul>
    <li class="current-menu-item"><a href="index.html">Start here</a></li>

    <li><a href="#">ReadMes</a>
    <ul>
    <li><a href="ms-data-peak-list-formats.html">MS peak list files</a></li>
    <li><a href="using-search-engines.html">Using MS search engines</a></li>
    <li><a href="quantitation.html">Spectral counts and quantitation</a></li>
    <li><a href="gene-expression-omnibus.html">Gene Expression Omnibus</a></li>
    <li><a href="biological-interpretation.html">Network and functional analyses</a>
    <li><a href="repositories-and-tools.html">Links to online resources and tools</a>
    </ul>



    <li><a href="#">Proteomics case studies</a>
    <ul>
    <li><a href="ms-study-1.html">DNA Methyltransferase I (DNMT1) interactions</a></li>
    <li><a href="ms-study-7.html">DNMT1 mutant vs wild-type comparison</a></li>
    <li><a href="ms-study-5.html">Proteasome and EIF complexes</a></li>
    <li><a href="ms-study-3.html">A DNA-damage protein complex</a></li>
    <li><a href="ms-study-6.html">Embryonic stem cell proteomes</a></li>
    <li><a href="label-free-studies.html">Label-free quantification</a></li>
    <li><a href="label-free-using-Scaffold.html">Label-free quant using Scaffold 1</a></li>
    <li><a href="label-free-using-Scaffold-2.html">Label-free quant using Scaffold 2</a></li>
    </ul>
    </li>


    <li><a href="#">Transcriptomics case studies</a>
    <ul>
    <li><a href="geo-case-studies.html">Gene Expression Omnibus (GEO) case studies</a></li>
    </ul>
         
       
    </ul>
    </li>



    </ul>

    </nav>
    <br>
    <br>

<!-- navigation bar and head END-->

































































<h3>Protein Quantitation using spectral counts</h3>
<p>Although identification of proteins from biological samples provides important information, in many cases, we need to know how the abundance of peptides and proteins differs between samples. In many cases, the identified proteins from biological samples to be compared will be similar, but their relative levels may be different, and this may be important for the underlying biology.</p>
<p class="vspace">There are several different methods for peptide and protein quantitation in mass spectrometry proteomics experiments, some requiring specialized analytical protocols such as isotope-labeling. Here we look at a &quot;label-free&quot; method for quantitation that can be applied to shotgun proteomics experiments. This method, known as &quot;spectral counting&quot; uses the number of times that a given peptide is sequenced by the mass spectrometer as an indicator of the abundance of that peptide. Thus by summing these counts for each peptide, a total count per protein can be derived, and these can be compared across biological samples</p>
<table border="0">
  <tbody>
    <tr>
<td><a class="urllink" href="http://www.proteomesoftware.com/Proteome_software_prod_Scaffold3_download-main.html" rel="nofollow"><img alt="" height="286" src="images/scaffold_screen.png" style="vertical-align: baseline;" width="400" /></a></td>
      <td>
	<ul>
	  <li>Scaffold software computes peptide counts, and provides side-by-side view of total counts per protein across samples, making it a simple and convenient way of identifying differences in protein abundance across samples.
	  <li>If you have not already done so, download and install <a class="urllink" href="http://www.proteomesoftware.com/Proteome_software_prod_Scaffold3_download-main.html" rel="nofollow">Scaffold</a>.</li>
	  <li><span class="wikilink"><a href="label-free-using-Scaffold.html">Label-free case study 1 (see Quantification studies)</a></span> and <span class="wikilink"><a href="label-free-using-Scaffold-2.html">Label-free case study 2 (see Quantification studies)</a></span> provide example Scaffold files that demonstrate how spectral counts are used.</li>
	  </ul>
	</td>
      
      </tr>
    </tbody>
</table>






















































